py_gtfs_rt_ingestion/tests/test_convert.py

```diff
@@ -313,3 +314,315 @@ def test_bus_vehicle_positions_file_conversion() -> None:
             assert upper == str(np_df[col].max())
         if lower != "nan":
             assert lower == str(np_df[col].min())
+
+
+def test_schedule_conversion() -> None:
```

Comment:
i added a test zip file and ensured it was parsing things out the way I expected. field names came from mbta's github and lenghts were found via inspection.